client/blocks/plugins-update-manager/index.tsx

```diff
@@ -18,7 +23,14 @@ interface Props {
 export const PluginsUpdateManager = ( props: Props ) => {
 	const { siteSlug, context, onNavBack, onCreateNewSchedule } = props;
 	const { data: schedules = [] } = useScheduleUpdatesQuery( siteSlug );
-	const hideCreateButton = schedules.length === MAX_SCHEDULES || schedules.length === 0;
+	const siteId = useSelector( getSelectedSiteId );
+	const hasScheduledUpdate = useSelector( ( state ) =>
```

Comment:
Can we call it `hasScheduledUpdatesFeature` to make it clear that the site doesn't have an update pending or anything?

Reply:
Sure thing!